src/usr/share/netcheck/netcheck-daemon.py

```diff
@@ -96,19 +96,22 @@ def sig_term_handler(signal, stack_frame):
     daemon_context = daemon.DaemonContext(
         working_directory='/',
         pidfile=pidlockfile.PIDLockFile(PID_FILE),
-        umask=0
+        # TODO: Set this back to 0 when NetCheck properly drops permissions.
+        umask=0o022
         )
 
     daemon_context.signal_map = {
         signal.SIGTERM: sig_term_handler
         }
 
     daemon_context.files_preserve = [log_file_handle]
+    broadcaster = parkbenchcommon.broadcaster.Broadcaster(
+        'NetCheck', 'network-reconnected', 0, 0)
```

Comment:
Can we use named parameters here. The meaning of the parameters are not obvious.